Unable to compile cam6_3_154 with nvhpc/24.3 on Derecho

### What happened?

When I checked out the CAM tag `cam6_3_154` with the `ccs_config0.0.99` tag, I failed to build a `F2000dev` test on Derecho. The error message in the `cesm.bldlog.*` file looks like:
```
Building a single executable version of target coupled model
ERROR: Command gmake exec_se -j 16 EXEC_SE=../../cesm.exe COMP_NAME=driver CIME_MODEL=cesm  SMP=FALSE CASEROOT="/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154" CASETOOLS="/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools" CIMEROOT="/glade/derecho/scratch/sunjian/cam6_3_154/cime" SRCROOT="/glade/derecho/scratch/sunjian/cam6_3_154" COMP_INTERFACE="nuopc" COMPILER="nvhpc" DEBUG="FALSE" EXEROOT="/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld" RUNDIR="/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/run" INCROOT="/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include" LIBROOT="/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib" MACH="derecho" MPILIB="mpich" NINST_VALUE="c1a1l1i1o1r1" OS="CNL" PIO_VERSION=2 SHAREDLIBROOT="/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld" BUILD_THREADED="FALSE" USE_ESMF_LIB="TRUE" USE_MOAB="FALSE" CAM_CONFIG_OPTS="-phys cam_dev  -rad rrtmg  -pcols 16" COMP_ATM="cam" COMP_ICE="cice" COMP_GLC="sglc" COMP_LND="clm" COMP_OCN="docn" COMP_ROF="mosart" COMP_WAV="swav" USE_TRILINOS="FALSE" USE_ALBANY="FALSE" USE_PETSC="FALSE"  GLC_PRESENT=FALSE WAV_PRESENT=FALSE ESP_PRESENT=FALSE IAC_PRESENT=FALSE -f /glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile  failed rc=2
out=/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/mkSrcfiles
cp -f Filepath Deppath
/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/mkDepends  Deppath Srcfiles > Depends
ftn -c -I. -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/fox/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/dshr -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/c1a1l1i1o1r1/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/finclude -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/zlnp/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/atm/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ice/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ocn/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/glc/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/rof/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/wav/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/esp/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/iac/obj -I -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/clm/obj -I. -I/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/SourceMods/src.drv -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include -O -Mnofma -i4 -gopt  -time -Mextend -byteswapio -Mflushz -Kieee -target=zen3 -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-c/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/3c7o/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-fortran/4.6.1/cray-mpich/8.1.27/nvhpc/24.3/i6rj/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallelio/2.6.2/cray-mpich/8.1.27/nvhpc/24.3/dcds/include  -DCNL  -DCESMCOUPLED -DFORTRANUNDERSCORE -DNO_SHR_VMATH -DNO_R16  -DCPRPGI -DLINUX -DHAVE_GETTID -DNDEBUG -DUSE_ESMF_LIB -DHAVE_MPI -DNUOPC_INTERFACE -DPIO2 -DHAVE_SLASHPROC -D_PNETCDF -DESMF_VERSION_MAJOR=8 -DESMF_VERSION_MINOR=6 -DATM_PRESENT -DICE_PRESENT -DLND_PRESENT -DOCN_PRESENT -DROF_PRESENT -DMED_PRESENT -DPIO2 -Mfree -DUSE_CONTIGUOUS= /glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator/med_kind_mod.F90
ftn -c -I. -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/fox/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/dshr -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/c1a1l1i1o1r1/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/finclude -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/zlnp/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/atm/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ice/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ocn/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/glc/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/rof/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/wav/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/esp/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/iac/obj -I -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/clm/obj -I. -I/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/SourceMods/src.drv -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include -O -Mnofma -i4 -gopt  -time -Mextend -byteswapio -Mflushz -Kieee -target=zen3 -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-c/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/3c7o/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-fortran/4.6.1/cray-mpich/8.1.27/nvhpc/24.3/i6rj/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallelio/2.6.2/cray-mpich/8.1.27/nvhpc/24.3/dcds/include  -DCNL  -DCESMCOUPLED -DFORTRANUNDERSCORE -DNO_SHR_VMATH -DNO_R16  -DCPRPGI -DLINUX -DHAVE_GETTID -DNDEBUG -DUSE_ESMF_LIB -DHAVE_MPI -DNUOPC_INTERFACE -DPIO2 -DHAVE_SLASHPROC -D_PNETCDF -DESMF_VERSION_MAJOR=8 -DESMF_VERSION_MINOR=6 -DATM_PRESENT -DICE_PRESENT -DLND_PRESENT -DOCN_PRESENT -DROF_PRESENT -DMED_PRESENT -DPIO2 -Mfree -DUSE_CONTIGUOUS= /glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn/shr_flux_mod.F90
ftn -c -I. -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/fox/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/dshr -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/c1a1l1i1o1r1/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/finclude -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/zlnp/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/atm/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ice/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ocn/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/glc/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/rof/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/wav/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/esp/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/iac/obj -I -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/clm/obj -I. -I/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/SourceMods/src.drv -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include -O -Mnofma -i4 -gopt  -time -Mextend -byteswapio -Mflushz -Kieee -target=zen3 -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-c/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/3c7o/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-fortran/4.6.1/cray-mpich/8.1.27/nvhpc/24.3/i6rj/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallelio/2.6.2/cray-mpich/8.1.27/nvhpc/24.3/dcds/include  -DCNL  -DCESMCOUPLED -DFORTRANUNDERSCORE -DNO_SHR_VMATH -DNO_R16  -DCPRPGI -DLINUX -DHAVE_GETTID -DNDEBUG -DUSE_ESMF_LIB -DHAVE_MPI -DNUOPC_INTERFACE -DPIO2 -DHAVE_SLASHPROC -D_PNETCDF -DESMF_VERSION_MAJOR=8 -DESMF_VERSION_MINOR=6 -DATM_PRESENT -DICE_PRESENT -DLND_PRESENT -DOCN_PRESENT -DROF_PRESENT -DMED_PRESENT -DPIO2 -Mfree -DUSE_CONTIGUOUS= /glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver/esm_time_mod.F90
ftn -c -I. -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/fox/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/dshr -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/c1a1l1i1o1r1/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/finclude -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/zlnp/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/atm/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ice/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ocn/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/glc/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/rof/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/wav/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/esp/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/iac/obj -I -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/clm/obj -I. -I/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/SourceMods/src.drv -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include -O -Mnofma -i4 -gopt  -time -Mextend -byteswapio -Mflushz -Kieee -target=zen3 -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-c/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/3c7o/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-fortran/4.6.1/cray-mpich/8.1.27/nvhpc/24.3/i6rj/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallelio/2.6.2/cray-mpich/8.1.27/nvhpc/24.3/dcds/include  -DCNL  -DCESMCOUPLED -DFORTRANUNDERSCORE -DNO_SHR_VMATH -DNO_R16  -DCPRPGI -DLINUX -DHAVE_GETTID -DNDEBUG -DUSE_ESMF_LIB -DHAVE_MPI -DNUOPC_INTERFACE -DPIO2 -DHAVE_SLASHPROC -D_PNETCDF -DESMF_VERSION_MAJOR=8 -DESMF_VERSION_MINOR=6 -DATM_PRESENT -DICE_PRESENT -DLND_PRESENT -DOCN_PRESENT -DROF_PRESENT -DMED_PRESENT -DPIO2 -Mfree -DUSE_CONTIGUOUS= /glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver/t_driver_timers_mod.F90
ftn -c -I. -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/fox/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/CDEPS/dshr -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/c1a1l1i1o1r1/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/finclude -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/zlnp/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/atm/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ice/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/ocn/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/glc/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/rof/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/wav/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/esp/obj -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/iac/obj -I -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/include -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/nvhpc/mpich/nodebug/nothreads/nuopc/nuopc/esmf/clm/obj -I. -I/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/SourceMods/src.drv -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../mediator -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/flux_atmocn -I/glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver -I/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/lib/include -O -Mnofma -i4 -gopt  -time -Mextend -byteswapio -Mflushz -Kieee -target=zen3 -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/esmf/8.6.0/cray-mpich/8.1.27/nvhpc/24.3/2u4m/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-c/4.9.2/cray-mpich/8.1.27/nvhpc/24.3/3c7o/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/netcdf-fortran/4.6.1/cray-mpich/8.1.27/nvhpc/24.3/i6rj/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallel-netcdf/1.12.3/cray-mpich/8.1.27/nvhpc/24.3/jqjr/include -I/glade/u/apps/derecho/23.09/spack/opt/spack/parallelio/2.6.2/cray-mpich/8.1.27/nvhpc/24.3/dcds/include  -DCNL  -DCESMCOUPLED -DFORTRANUNDERSCORE -DNO_SHR_VMATH -DNO_R16  -DCPRPGI -DLINUX -DHAVE_GETTID -DNDEBUG -DUSE_ESMF_LIB -DHAVE_MPI -DNUOPC_INTERFACE -DPIO2 -DHAVE_SLASHPROC -D_PNETCDF -DESMF_VERSION_MAJOR=8 -DESMF_VERSION_MINOR=6 -DATM_PRESENT -DICE_PRESENT -DLND_PRESENT -DOCN_PRESENT -DROF_PRESENT -DMED_PRESENT -DPIO2 -Mfree -DUSE_CONTIGUOUS= /glade/derecho/scratch/sunjian/cam6_3_154/components/cmeps/cime_config/../cesm/driver/util.F90
-I<incdir>          Add directory to include file search path
-I<incdir>          Add directory to include file search path
-I<incdir>          Add directory to include file search path
-I<incdir>          Add directory to include file search path
-I<incdir>          Add directory to include file search path
err=cat: Srcfiles: No such file or directory
Warning: -target=zen3 option ignored.
Warning: -target=zen3 option ignored.
Warning: -target=zen3 option ignored.
Warning: -target=zen3 option ignored.
Warning: -target=zen3 option ignored.
nvfortran-Error-Switch -I expects an argument
gmake: *** [/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile:1001: med_kind_mod.o] Error 1
gmake: *** Waiting for unfinished jobs....
nvfortran-Error-Switch -I expects an argument
gmake: *** [/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile:1001: shr_flux_mod.o] Error 1
nvfortran-Error-Switch -I expects an argument
gmake: *** [/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile:1001: t_driver_timers_mod.o] Error 1
nvfortran-Error-Switch -I expects an argument
nvfortran-Error-Switch -I expects an argument
gmake: *** [/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile:1001: esm_time_mod.o] Error 1
gmake: *** [/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/Tools/Makefile:1001: util.o] Error 1
```
This error happens because there is an option `-I` without a path appended to it somehow.

When I checked out the CAM tag `cam6_3_153` with the `ccs_config0.0.99` tag, I was able to build the same `F2000dev` case on Derecho successfully.

### What are the steps to reproduce the bug?

./create_newcase --case /path_to_case --mach derecho --res ne30pg3_ne30pg3_mg17 --compset F2000dev --compiler nvhpc --ngpus-per-node 4 --gpu-type a100 --gpu-offload openacc
cd /path_to_case 
./case.setup
./case.build

### What CAM tag were you using?

cam6_3_154

### What machine were you running CAM on?

CISL machine (e.g. cheyenne)

### What compiler were you using?

NVHPC

### Path to a case directory, if applicable

/glade/derecho/scratch/sunjian/cam6/case/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154

### Will you be addressing this bug yourself?

No

### Extra info

The cesm error log for cam6_3_154 could be found on Derecho at `/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_154/bld/cesm.bldlog.240503-115432`.

The successful build of cam6_3_153 could be found on Derecho at `/glade/derecho/scratch/sunjian/cam6_run/F2000dev.ne30pg3_ne30pg3_mg17.derecho.nvhpc.gpu04_pcols00016_mpi0128_thread001_rrtmg.aer_bug_153`.

--- a/Externals.cfg
+++ b/Externals.cfg
@@ -49,6 +49,13 @@ repo_url = https://github.com/ESCOMP/CESM_share
 local_path = share
 required = True
 
+[mct]
+tag = MCT_2.11.0
+protocol = git
+repo_url = https://github.com/MCSclimate/MCT
+local_path = libraries/mct
+required = True
+
 [parallelio]
 tag = pio2_6_2
 protocol = git

--- a/Externals.cfg
+++ b/Externals.cfg
@@ -1,5 +1,5 @@
 [ccs_config]
-tag = ccs_config_cesm0.0.85
+tag = ccs_config_cesm0.0.106
 protocol = git
 repo_url = https://github.com/ESMCI/ccs_config_cesm
 local_path = ccs_config
@@ -13,7 +13,7 @@ local_path = components/cice5
 required = True
 
 [cice6]
-tag = cesm_cice6_4_1_10
+tag = cesm_cice6_5_0_7
 protocol = git
 repo_url = https://github.com/ESCOMP/CESM_CICE
 local_path = components/cice
@@ -21,14 +21,14 @@ externals = Externals.cfg
 required = True
 
 [cmeps]
-tag = cmeps0.14.59
+tag = cmeps0.14.60
 protocol = git
 repo_url = https://github.com/ESCOMP/CMEPS.git
 local_path = components/cmeps
 required = True
 
 [cdeps]
-tag = cdeps1.0.31
+tag = cdeps1.0.33
 protocol = git
 repo_url = https://github.com/ESCOMP/CDEPS.git
 local_path = components/cdeps
@@ -36,7 +36,7 @@ externals =  Externals_CDEPS.cfg
 required = True
 
 [cpl7]
-tag = cpl77.0.7
+tag = cpl77.0.8
 protocol = git
 repo_url = https://github.com/ESCOMP/CESM_CPL7andDataComps
 local_path = components/cpl7
@@ -47,13 +47,6 @@ tag = share1.0.18
 protocol = git
 repo_url = https://github.com/ESCOMP/CESM_share
 local_path = share
-required = True
-
-[mct]
-tag = MCT_2.11.0
-protocol = git
-repo_url = https://github.com/MCSclimate/MCT
-local_path = libraries/mct
 required = True
 
 [parallelio]
@@ -71,7 +64,7 @@ local_path = cime
 required = True
 
 [cism]
-tag = cismwrap_2_1_96
+tag = cismwrap_2_1_100
 protocol = git
 repo_url = https://github.com/ESCOMP/CISM-wrapper
 local_path = components/cism
@@ -79,7 +72,7 @@ externals = Externals_CISM.cfg
 required = True
 
 [clm]
-tag = ctsm5.1.dev145
+tag = ctsm5.2.005
 protocol = git
 repo_url = https://github.com/ESCOMP/CTSM
 local_path = components/clm
@@ -95,14 +88,14 @@ externals = Externals_FMS.cfg
 required = True
 
 [mosart]
-tag = mosart1_0_48
+tag = mosart1_0_49
 protocol = git
 repo_url = https://github.com/ESCOMP/MOSART
 local_path = components/mosart
 required = True
 
 [rtm]
-tag = rtm1_0_78
+tag = rtm1_0_79
 protocol = git
 repo_url = https://github.com/ESCOMP/RTM
 local_path = components/rtm

--- a/cime_config/testdefs/testlist_cam.xml
+++ b/cime_config/testdefs/testlist_cam.xml
@@ -73,7 +73,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
 
@@ -83,7 +83,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
 
@@ -93,7 +93,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam_short"/>
     </machines>
     <options>
-      <option name="wallclock">00:20:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
 
@@ -1811,7 +1811,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:90:00</option>
       <option name="comment">CAM7 dev low top ~40 km</option>
     </options>
   </test>
@@ -1829,7 +1829,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:20:00</option>
+      <option name="wallclock">00:60:00</option>
       <option name="comment">CAM7 dev mid top ~80 km</option>
     </options>
   </test>
@@ -2033,7 +2033,7 @@
       <machine name="derecho" compiler="intel" category="camchem_hco"/>
     </machines>
     <options>
-      <option name="wallclock">00:20:00</option>
+      <option name="wallclock">00:60:00</option>
       <option name="comment">FCSD CAM-chem f19 with HEMCO exact restart/PE tests</option>
     </options>
   </test>
@@ -2390,7 +2390,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:30:00</option>
     </options>
   </test>
   <test compset="FWHIST_BGC" grid="f09_f09_mg17" name="ERP_Ld3" testmods="cam/reduced_hist1d">
@@ -2415,7 +2415,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:40:00</option>
+      <option name="wallclock">00:80:00</option>
     </options>
   </test>
   <test compset="FWHIST" grid="f09_f09_mg17" name="SMS_D_Ln9" testmods="cam/reduced_hist3s">
@@ -2424,7 +2424,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:20:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
   <test compset="FX2000" grid="f19_f19_mg17" name="SMS_D_Ln9"   testmods="cam/outfrq9s">
@@ -2450,7 +2450,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
   <test compset="FXHIST" grid="f19_f19_mg17" name="SMS_D_Ln9" testmods="cam/outfrq9s_amie">
@@ -2459,7 +2459,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:30:00</option>
     </options>
   </test>
   <test compset="QPX2000" grid="f19_f19_mg17" name="SMS_D_Ln9_P240x3" testmods="cam/outfrq9s_ltr">
@@ -2725,7 +2725,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:60:00</option>
     </options>
   </test>
   <test compset="FWscHIST" grid="f09_f09_mg17" name="ERP_Ld3" testmods="cam/outfrq3d">
@@ -2748,7 +2748,7 @@
       <machine name="derecho" compiler="intel" category="ghg_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:30:00</option>
       <option name="comment">Checks that exact restarts occur when crossing the December 16th date boundary with WACCM-SC chemistry in low top configuration</option>
     </options>
   </test>
@@ -2800,7 +2800,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:10:00</option>
+      <option name="wallclock">00:30:00</option>
       <option name="comment">WACCM w/ RRTMGP</option>
     </options>
   </test>

--- a/cime_config/testdefs/testlist_cam.xml
+++ b/cime_config/testdefs/testlist_cam.xml
@@ -73,7 +73,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:10:00</option>
     </options>
   </test>
 
@@ -83,7 +83,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:10:00</option>
     </options>
   </test>
 
@@ -93,7 +93,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam_short"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:20:00</option>
     </options>
   </test>
 
@@ -1811,7 +1811,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:90:00</option>
+      <option name="wallclock">00:30:00</option>
       <option name="comment">CAM7 dev low top ~40 km</option>
     </options>
   </test>
@@ -1829,7 +1829,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:20:00</option>
       <option name="comment">CAM7 dev mid top ~80 km</option>
     </options>
   </test>
@@ -2033,7 +2033,7 @@
       <machine name="derecho" compiler="intel" category="camchem_hco"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:20:00</option>
       <option name="comment">FCSD CAM-chem f19 with HEMCO exact restart/PE tests</option>
     </options>
   </test>
@@ -2390,7 +2390,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:10:00</option>
     </options>
   </test>
   <test compset="FWHIST_BGC" grid="f09_f09_mg17" name="ERP_Ld3" testmods="cam/reduced_hist1d">
@@ -2415,7 +2415,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:80:00</option>
+      <option name="wallclock">00:40:00</option>
     </options>
   </test>
   <test compset="FWHIST" grid="f09_f09_mg17" name="SMS_D_Ln9" testmods="cam/reduced_hist3s">
@@ -2424,7 +2424,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:20:00</option>
     </options>
   </test>
   <test compset="FX2000" grid="f19_f19_mg17" name="SMS_D_Ln9"   testmods="cam/outfrq9s">
@@ -2450,7 +2450,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:30:00</option>
     </options>
   </test>
   <test compset="FXHIST" grid="f19_f19_mg17" name="SMS_D_Ln9" testmods="cam/outfrq9s_amie">
@@ -2459,7 +2459,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:10:00</option>
     </options>
   </test>
   <test compset="QPX2000" grid="f19_f19_mg17" name="SMS_D_Ln9_P240x3" testmods="cam/outfrq9s_ltr">
@@ -2725,7 +2725,7 @@
       <machine name="derecho" compiler="intel" category="prealpha"/>
     </machines>
     <options>
-      <option name="wallclock">00:60:00</option>
+      <option name="wallclock">00:10:00</option>
     </options>
   </test>
   <test compset="FWscHIST" grid="f09_f09_mg17" name="ERP_Ld3" testmods="cam/outfrq3d">
@@ -2748,7 +2748,7 @@
       <machine name="derecho" compiler="intel" category="ghg_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:10:00</option>
       <option name="comment">Checks that exact restarts occur when crossing the December 16th date boundary with WACCM-SC chemistry in low top configuration</option>
     </options>
   </test>
@@ -2800,7 +2800,7 @@
       <machine name="derecho" compiler="intel" category="aux_cam"/>
     </machines>
     <options>
-      <option name="wallclock">00:30:00</option>
+      <option name="wallclock">00:10:00</option>
       <option name="comment">WACCM w/ RRTMGP</option>
     </options>
   </test>

--- a/cime_config/testdefs/testlist_cam.xml
+++ b/cime_config/testdefs/testlist_cam.xml
@@ -1444,7 +1444,7 @@
       <machine name="izumi" compiler="nag" category="prealpha"/>
     </machines>
   </test>
-  <test compset="FGRAYRAD" grid="f09_f09_mg17" name="ERP_D_Ln9_Vmct" testmods="cam/outfrq9s">
+  <test compset="FGRAYRAD" grid="f09_f09_mg17" name="ERP_D_Ln9" testmods="cam/outfrq9s">
     <machines>
       <machine name="izumi" compiler="nag" category="prebeta"/>
     </machines>

--- a/cime_config/testdefs/testlist_cam.xml
+++ b/cime_config/testdefs/testlist_cam.xml
@@ -1444,7 +1444,7 @@
       <machine name="izumi" compiler="nag" category="prealpha"/>
     </machines>
   </test>
-  <test compset="FGRAYRAD" grid="f09_f09_mg17" name="ERP_D_Ln9" testmods="cam/outfrq9s">
+  <test compset="FGRAYRAD" grid="f09_f09_mg17" name="ERP_D_Ln9_Vmct" testmods="cam/outfrq9s">
     <machines>
       <machine name="izumi" compiler="nag" category="prebeta"/>
     </machines>

